### PR TITLE
Support for solid background resolution adjustment

### PR DESCRIPTION
**Pull request by Abdillah**
Allow background size to cover way-cooler's screen size.

It works only when `way-cooler-bg` started, it calculates background resolution and hands it to `way-cooler`. Yet, it doesn't support recalculation. So, when resolution changes, the background got messy again.

### CODE REVIEW

**Comment by Abdillah** [comment]
As an additional note, when `way-cooler` stopped before `way-cooler-bg` does, it panicked with "'Connection with the compositor was lost. ... Broken pipe" which is *natural*. Partly because my wrong `init.lua` setup.

File: src/main.rs [hunk 1]
```diff
@@ -89,6 +95,49 @@ fn rgba_conversion(num: u8, third_num: u32) -> u8 {
...
+            let (width, height) = (
+                (&items[0]).inner::<u32>()
+                    .expect("Way Cooler returned an unexpected value"),
+                (&items[1]).inner::<u32>()
+                    .expect("Way Cooler returned an unexpected value")
+            );
+            println!("{:?}, {:?}", width, height);
+            (width as i32, height as i32)
+        },
+        _ => panic!("Colud not get resolution of screen")
```
Comment (Timidger) [inline src/main.rs]: Typo, should be "Could"

**Comment by Timidger** [comment]
Yeah, mine has done the same thing as well. I don't see this is a problem that it returns a non-zero exit status, but this might be amended it later if it messes with some user's script.

File: src/main.rs (hunk 1 above)
Comment (Abdillah) [reply src/main.rs]: Fixed

**Review by Timidger — APPROVED** [review]

**Comment by Timidger** [comment]
Looks good, thanks for the patch!

**Pull request merged**